falcon/constants.py

```diff
@@ -10,6 +11,19 @@
     'TRACE',
 )
 
+# RFC 3253 methods
+WEBDAV_METHODS = (
```

Comment:
Just to be on the safe side, I'd like to see some tests for one or two of these methods. The tests should check that a resource can successfully implement responders for these, and also that a request for one of these methods triggers the default `405 HTTPMethodNotAllowed` responder when the method isn't implemented for a specific resource.

Reply:
I think i've addressed this now with updates to `test_http_method_routing`.

`on_report() was implemented in `ThingsResource`, with a test to exercise the route, and a few of the test_method_not_allowed methods were updated to also iterate through the webdav methods.